doc/guides/compile.md

```diff
@@ -338,6 +338,47 @@ When debugging mode is enabled
   * Because `-g` flag would be added to `mrbc` runner.
     * You can have better backtrace of mruby scripts with this.
 
+### Additional license terms
+
+The license terms set in the `conf.terms` property are printed as part of
+the `<build-dir>/LICENSE.yml` file.
+This initial value is an empty array (`[]`). If you want to add it, do the
+following:
+
+```ruby
+conf.terms << "path/to/ADDITIONAL-LICENSE"
+```
+
+The file path can be specified as an absolute path as well as a relative
+path based on the build directory.
+Also, you can give not only the file path but also a hash that combines
+the name and the content of the terms.
+
+Objects that can be specified directly in `conf.terms` are `nil`, strings
+that represent paths, array, hash, plus objects that have both `#to_path`
+and `#read` methods (assuming file and pathname).
+
+Inside the array, you can add anything that is given directly to
+`conf.terms`.
+
+In hash, you can specify the object that represents the path (name) as a
+key and the object to be the content as a set.
+The object that represents a path is a string or an object that has a
+`#to_path` method (assuming file or pathname).
```

Comment:
Is symbol not supported for the key?

Reply:
I don't intend to support symbols at this time as I think the key in this case should mean the pathname.
I was curious, and when I tried combining `Pathname` and the symbol in CRuby, I got an error here as well.

```console
% ruby30 -rpathname -e 'p Pathname("/path/to") + :name'
/usr/local/lib/ruby/3.0/pathname.rb:353:in `initialize': no implicit conversion of Symbol into String (TypeError)
```